apps/remix-ide/src/app.js

```diff
@@ -368,13 +368,13 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   // Ask to opt in to Matomo for remix, remix-alpha and remix-beta
   if (window.location.hostname.includes('.ethereum.org') && !registry.get('config').api.exists('settings/matomo-analytics')) {
     modalDialog(
-      'Help us to improve our IDE',
+      'Help us to improve Remix IDE',
       yo`
       <div>
-        <p>Remix IDE uses <a href="https://matomo.org" target="_blank">Matomo</a>, an open source data analytics platform, to improve our website. Matomo on Remix is opt-in - meaning that we won't collect any information unless you agree.</p>
-        <p>We realize that our users have sensitive information in their code and that the privacy of our users must be protected.</p>
+        <p>An Opt-in version of <a href="https://matomo.org" target="_blank">Matomo</a>, an open source data analytics platform is being used to improve the Remix IDE website.</p>
```

Comment:
I would choose `Remix IDE platform` instead of `Remix IDE website`

Reply:
Or just Remix IDE